portality/settings.py

```diff
@@ -266,6 +267,16 @@
             }
         }
     },
+    "str": {
```

Comment:
does this mean we now have both `str` and `unicode` in our mappings? That strikes me as a bad time.

Reply:
yes, I added this because a "str" coerce type was added to the data objects.  I think we decided to back this out, so this can go.  I'll leave it in for the moment, until we've tracked that other change through.